packages/quantic/force-app/main/default/lwc/quanticResultQuickview/e2e/pageObject.ts

```diff
@@ -0,0 +1,39 @@
+import type {Locator, Page} from '@playwright/test';
+
+export class ResultQuickviewObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get button(): Locator {
+    return this.page.locator('c-quantic-result-quickview button');
+  }
+
+  get buttonIcon(): Locator {
+    return this.button.locator('lightning-icon');
+  }
```

Comment:
```suggestion
  get quickviewButton(): Locator {
    return this.page.locator('c-quantic-result-quickview button');
  }

  get quickviewButtonIcon(): Locator {
    return this.quickviewButton.locator('lightning-icon');
  }
```

Reply:
I would rename them quickviewButton, etc. to make it more precise and clear what we are selecting here.

Reply:
Je suis pas tant d'accord vue que l'objet est déja un `ResultQuickviewObject`, dans les tests `resultQuickview.button` look mieux que `resultQuickview.quickviewButton` ?

Reply:
c'est un matter de preference pas un deal breaker pour moi, dans les autres test files c'est le pattern qu'on follow